brave/brave-ios
# Fix #5216: Wallet Panel Integration

## Summary of Changes
Integrated wallet panel:
1. account selection
2. account balance
3. buy/send/swap
4. transaction history
5. full screen button
6. settings button
7. background color

This pull request fixes #5216 

## Submitter Checklist:


## Test Plan:
1. Connect a dapp with your brave wallet
2. The wallet panel should show up
3. (If you already connected, tapping the wallet icon in the url bar should also show the wallet panel)
4. Check if current selected account is being displayed 
8. Check if selected account native token's balance is correct 
9. Click the account icon will bring up an account selection screen, you can change account by tapping an account row, or adding a new account by clicking the `+` button
10. Once you change an account check if it reflects back to the Wallet Panel
11. Click the two-arrow button will bring up Buy/Send/Swap option at the bottom just like the regular flow inside Wallet
12. Make some transactions
13. Click the clock button will bring up a screen with all transaction with the current account within the current network. 
14. If you made some transaction in step 9, all the pending transactions should show up here too.


## Screenshots:
![simulator_screenshot_59D4C3AC-2DE0-428B-BCA5-79362DB752A6](https://user-images.githubusercontent.com/1187676/162859867-c04a1a2c-0336-4e92-8592-3c2169b11d14.png) | ![simulator_screenshot_D6D02DA7-E3FB-4F36-86A2-EFE90E62C344](https://user-images.githubusercontent.com/1187676/161853363-afdcd4e4-9477-4243-9121-4cc8ae96107a.png) | ![simulator_screenshot_868D88F4-FE4F-4126-A65B-AF682880D0FA](https://user-images.githubusercontent.com/1187676/161853417-c61e0ca7-c965-42e7-b504-ed514ef25925.png) | ![simulator_screenshot_2D0381F5-66AB-40B0-95B2-C5D1C182CB8C](https://user-images.githubusercontent.com/1187676/161853449-cf5111c1-be4f-4359-bf89-5e201bf88660.png)
--|--|--|--

## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
